driver-core/src/main/java/com/datastax/driver/core/JdkSSLOptions.java

```diff
@@ -52,7 +53,12 @@ protected JdkSSLOptions(SSLContext context, String[] cipherSuites) {
 
     @Override
     public SslHandler newSSLHandler(SocketChannel channel) {
-        SSLEngine engine = newSSLEngine(channel);
+        return newSSLHandler(channel, null);
```

Comment:
Agreed.

However your remark made me think of the following situation: If someone is extending `JdkSSLOptions` and overriding this method to customize the `SslHandler`, he will experience runtime problems because his overridden method will not be called anymore. Granted, users should override `newSSLEngine` rather than `newSSLHandler`, but it is not forbidden to do so. Maybe something worth mentioning in the upgrade guide?